console/android/GenericApp/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.openremote.app">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Should package attribute be removed?

Reply:
Yes, in newer android grade plugin, this is specified elsewhere